app/lib/frontend/templates.dart

```diff
@@ -984,5 +992,15 @@ final Map<String, Map> _logoData = const {
   },
 };
 
+String _formattedPlatformName(String platform) {
```

Comment:
This is also under-specified. An alternative could be to uppercase the platform names by default.

Reply:
That's something for @kevmoo